eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -65,12 +67,15 @@ import scala.util.{Failure, Success}
  *
  * Created by PM on 25/01/2016.
  *
- * @param datadir  directory where eclair-core will write/read its data.
- * @param seed_opt optional seed, if set eclair will use it instead of generating one and won't create a seed.dat file.
- * @param db       optional databases to use, if not set eclair will create the necessary databases
+ * @param datadir       directory where eclair-core will write/read its data.
+ * @param pluginParams  a list of (node feature bit, LN message tags) each network-enabled plugin is interested in.
```

Comment:
You shouldn't describe what's inside `PluginParams`, otherwise the comment will be obsolete and probably not fixed in we change the contents of `PluginParams`.

```suggestion
 * @param pluginParams  parameters for all configured plugins.
```